raiden-ts/src/channels/epics.ts

```diff
@@ -251,9 +251,69 @@ export function blockStaleEpic(
   );
 }
 
+function scanRegistryTokenNetworks({
+  address,
+  provider,
+  registryContract,
+  contractsInfo,
+  getTokenNetworkContract,
+}: RaidenEpicDeps): Observable<tokenMonitored> {
+  const encodedAddress = defaultAbiCoder.encode(['address'], [address]);
+  return getLogsByChunk$(provider, {
+    ...registryContract.filters.TokenNetworkCreated(null, null),
+    fromBlock: contractsInfo.TokenNetworkRegistry.block_number,
+    toBlock: provider.blockNumber,
+  }).pipe(
+    map(
+      logToContractEvent<[token: Address, token_network: Address, event: Event]>(registryContract),
+    ),
+    filter(isntNil),
+    filter(([, tokenNetwork]) => !!tokenNetwork),
+    toArray(),
+    mergeMap((logs) => {
+      if (!logs.length) return EMPTY;
+      const firstBlock = logs[0][2].blockNumber;
```

Comment:
A comment for what `[2]`is would be nice for the future.

Reply:
Fortunately, TS merged recently named tuple parameters, and I've already used it here, on code, no need for comment: if you inspect the `logs` type, you'll see something like `logs: [token: Address, token_network: Address, event: Event][]`, which tells you `[0]` is the first array element, and `[2]` is the `event: Event` member, which is the `ethers`'s `Event` object.